AzureCalling/External/Calling/AudioDeviceSelectionManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import AVFoundation
+
+class AudioDeviceSelectionManager {
```

Comment:
I think it would be better if the inputs/outputs to this class are purely enum values or IDs instead of a data model. The ViewController can handle mapping to specific images and selected state in the UI.